styles/base/_padding.scss

```diff
@@ -0,0 +1,220 @@
+// paddding scale
+.pad-1 {
+  padding: 3rem;
+}
+
+.pad-2 {
+  padding: 2.5rem;
+}
+
+.pad-3 {
+  padding: 2rem;
+}
+
+.pad-4 {
+  padding: 1.5rem;
+}
+
+.pad-5 {
+  padding: 1rem;
+}
+
+.pad-6 {
+  padding: .75rem;
+}
+
+.pad-7 {
+  padding: .5rem;
```

Comment:
Would be nice we all utility classes are written on one line  this way it's easy to scan them quickly (because of repetition).